### ISSUE TITLE
Installing microsoft.gpg leaves the trusted key user owned

### ISSUE THREAD

**Issue by jkesanen** [microsoft/ProcDump-for-Linux#45]
### Expected behavior

When following the installation instructions for `1. Add the Microsoft Product feed`, the trusted key is securely placed into the system's keyring.

### Actual behavior

The **microsoft.gpg** key is left user owned.

### Steps to reproduce the behavior
1. Add the Microsoft Product feed: curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
2. sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
3. ls -l /etc/apt/trusted.gpg.d/microsoft.gpg

### System information (e.g., distro, kernel version, etc.)

Ubuntu GNU/Linux 16.4.3 LTS.

**Comment by josalem** [comment]
Took a look into this.  The most up to date way of adding the product feed (copy/paste from https://dot.net) is:
```
wget -q https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb
```

You should be able to plug in 14.04, 16.04, or 18.04 into the URL.  The documentation needs to be updated in this repo to reflect this.

**Comment by jahabibi** [comment]
This has been resolved with #69 that includes updated installation instructions utilizing the packages-microsoft-prod.deb/rpm to install the GPG key and configure the Microsoft package repository.
